desktop/src/main/java/bisq/desktop/main/support/dispute/DisputeView.java

```diff
@@ -222,6 +263,12 @@ public void initialize() {
             showFullReport();
         });
 
+        sigCheckButton = new AutoTooltipButton(Res.get("support.sigCheck.button"));
+        HBox.setHgrow(sigCheckButton, Priority.NEVER);
+        sigCheckButton.setOnAction(e -> {
```

Comment:
Wouldn't it be better to automatically check the signature and add the result to the dispute graphics somewhere? Also, the refundagent gets this message when a user opens a refund case, this message doesn't verify since there is a header from the user included
```
System message: Mediator's dispute summary:
Mediator's node address: localhost:3600
```
Without that header the it would verify. Perhaps we should also make that message autoverify by adding a `----HEADER----` and `----END HEADER----` separator?

Another, probably separate issue, the popup from double clicking a chat message cannot be closed nicely (this might be an old thing though), never tried double clicking the messages in the chat.

Reply:
The signature is not intended as check for the traders/arbitrator as that is anyway part of the p2p messaging (each msg gets checked if it is signed correctly by the sender). It is only intended for DAO voters.

Reply:
"Another, probably separate issue, the popup from double clicking a chat message cannot be closed nicely (this might be an old thing though), never tried double clicking the messages in the chat." -> can you make an issue for that. I want not to extend that PR further, already too much ;-)

Reply:
Indeed, will look at that separately.